### PR TITLE
deploy: revert changes made for 3.4.0 release

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

File: charts/ceph-csi-cephfs/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 apiVersion: v1
-appVersion: v3.4.0
+appVersion: v3.4-canary
 description: "Container Storage Interface (CSI) driver,
```
Comment (Madhu-1) [inline charts/ceph-csi-cephfs/Chart.yaml:4]: `version: 3.4.0-canary` should be `version: 3.4-canary`. its correct for rbd but for cephfs its not updated

**Review by Madhu-1 — DISMISSED** [review]

File: charts/ceph-csi-cephfs/Chart.yaml (hunk 1 above)
Comment (humblec) [reply charts/ceph-csi-cephfs/Chart.yaml:4]: ah... correcting it. Thanks!

Comment (humblec) [reply charts/ceph-csi-cephfs/Chart.yaml:4]: @Madhu-1 done.. thanks

**Review by Madhu-1 — APPROVED** [review]

**Review by yati1998 — APPROVED** [review]

**Pull request merged**